tutorials/text/usetex.py

```diff
@@ -51,6 +51,27 @@
       "font.family": "serif",
       "font.serif": ["Palatino"],
   })
+  # It's also possible to use the reduced notation by directly setting font.family:
+  plt.rcParams.update({
+    "text.usetex": True,
+    "font.family": "Helvetica"
+  })
+
+Currently, the supported fonts are:
+
+================= ============================================================
+family            fonts
```

Comment:
Worth mentioning this as generic-family? (I think it's better to be explicit all over our docs)

There's also a possibility of what we're calling fonts in the second columns being their own families (for example, 'Courier'), but 'fonts' seem an okay title.

Reply:
> Worth mentioning this as generic-family? (I think it's better to be explicit all over our docs)

Maybe? Currently we have `If the fonts are not specified, the Computer Modern fonts are used by default`. Do you think it is not enough?

Reply:
Sorry for getting back after so long, I came here to add a comment, realized this was still pending.
It may be enough, I was/am just being extra cautious by segregating generic families and family-names throughout docs (see [this](https://discourse.matplotlib.org/t/fonts-and-font-families/22132) for a valid confusion).